c2corg_ui/templates/helpers/view.html

```diff
@@ -497,3 +502,64 @@ <h3 class="heading closed show-phone" ng-click="mainCtrl.animateHeaderIcon($even
       
   % endif
 </%def>
+
+
+
+<%def name="photoswipe_gallery()">\
+  <!-- Root element of PhotoSwipe. Must have class pswp. -->
```

Comment:
thats what I did, but they did appear literally  like "## comment", maybe it is a special context.